qcfractal/tests/test_services.py

```diff
@@ -201,7 +201,7 @@ def test_service_torsiondrive_option_energy_upper_limit(torsiondrive_fixture):
     assert pytest.approx(0.0007991274441437338, abs=1e-6) == final_energies[(-60, )]
 
 
-@mark_slow
+@pytest.mark.slow
```

Comment:
Ah, I see this approach. Ok, should work!

Reply:
We can also extend it with the 3 functions in the plugin in the future.